superset-frontend/src/components/AsyncAceEditor/index.tsx

```diff
@@ -25,6 +25,7 @@ import {
 } from 'brace';
 import AceEditor, { IAceEditorProps } from 'react-ace';
 import { acequire } from 'ace-builds/src-noconflict/ace';
+import 'ace-builds/webpack-resolver';
```

Comment:
Without this we get a failed network request when opening up the edit modal in the CSS list view + no syntax highlighting (it's even uglier with `dev-server`):

<img width="1204" alt="image" src="https://github.com/apache/superset/assets/33317356/2060c2e7-bf6c-4105-996a-72aaeced9819">

After adding this, no more failed network request + CSS syntax highlighting works:

<img width="1206" alt="image" src="https://github.com/apache/superset/assets/33317356/8a73df2b-cece-4959-95c2-3546e03eea5d">

Adding this import was a recommendation in this thread, and seemed to be an ok fix: https://github.com/securingsincity/react-ace/issues/725